scripts/vscripts/ZombieReborn/Infect.lua

```diff
@@ -75,9 +77,10 @@ function Infect_PickMotherZombies()
     print("Player count: " .. iPlayerCount .. ", Mother Zombies Spawned: " .. iMotherZombieCount)
 end
 
-function Infect_OnRoundFreezeEnd()
-    local iMZSpawntimeMinimum = (Convars:GetInt("zr_infect_spawn_time_min") or iInfectSpawntimeMin)
-    local iMZSpawntimeMaximum = (Convars:GetInt("zr_infect_spawn_time_max") or iInfectSpawntimeMax)
+-- hook this to OnRoundStart in main script
+function Infect_OnRoundStart()
```

Comment:
These 2 lines got reverted when you solved the merge conflict. See https://github.com/Source2ZE/ZombieReborn/commit/f3899667b3ef54c8d074a7dd987ec95659a3df08